site/src/sphinx/server-access-log.rst

```diff
@@ -163,6 +174,119 @@ the condition.
 |                                                   | nor ``304``.                                   |
 +---------------------------------------------------+------------------------------------------------+
 
+.. _request-log:
+
+Retrieving values from RequestLog
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:api:`RequestLog` holds information about the request, so a user may want to write these values on his or
+her access log file. To write them in a simple way, ``%{variable}L`` token is provided with the following
+supported variable:
+
++-------------------------------+--------------------------------------------------------------------+
+| Name                          | Description                                                        |
++===============================+====================================================================+
+| ``method``                    | the HTTP method value of the request                               |
++-------------------------------+--------------------------------------------------------------------+
+| ``path``                      | the absolute path part of the HTTP request URI                     |
++-------------------------------+--------------------------------------------------------------------+
+| ``query``                     | the query part of the HTTP request URI                             |
++-------------------------------+--------------------------------------------------------------------+
+| ``requestStartTimeMillis``    | the time when the processing of the request started,               |
+|                               | in milliseconds since the epoch                                    |
++-------------------------------+--------------------------------------------------------------------+
+| ``requestDurationMillis``     | the duration that was taken to consume or produce the request      |
+|                               | completely, in milliseconds                                        |
++-------------------------------+--------------------------------------------------------------------+
+| ``requestDurationNanos``      | the duration that was taken to consume or produce the request      |
+|                               | completely, in nanoseconds                                         |
++-------------------------------+--------------------------------------------------------------------+
+| ``requestLength``             | the length of the request content                                  |
++-------------------------------+--------------------------------------------------------------------+
+| ``requestCause``              | the cause of request processing failure. The detail message of a   |
+|                               | ``Throwable`` or the class name of the cause if no message is      |
+|                               | specified.                                                         |
```

Comment:
How about `The class name of the cause and the detail message of it will be contained if exists.`?

Reply:
Oh, I missed fixing this description! Thanks!